packages/vuetify/src/components/VCheckbox/VSimpleCheckbox.ts

```diff
@@ -10,7 +10,7 @@ import Colorable from '../../mixins/colorable'
 import Themeable from '../../mixins/themeable'
 
 // Utilities
-import { mergeListeners } from '../../util/mergeData'
+import { mergeListeners, mergeClasses } from '../../util/mergeData'
```

Comment:
move under line 14, break into new lines and order alphabetically please:

https://www.notion.so/vuetify/Components-d863ad73563d43609c434f92b3e25f08#9127c5ebc666490eaa82bfe633e56dcb

Reply:
Resolved